dietstory-routing/registration/views.py

```diff
@@ -124,24 +134,30 @@ def post(self, request, *args, **kwargs):
                     account.save()
 
                     try:
-                        send_verification_email(account.email, account_activation_token.make_token(account))
+                        token = account_activation_token.make_token(account)
+                        logger.debug("Verification Token: {}".format(token))
+                        send_verification_email(account.email, token)
                     except IOError:
-                        print("Failed to send email.") #TODO: Logging
-
+                        logger.warn("[RV-1] Failed to send email to: {}".format(account.email))
```

Comment:
Why is this a warning when it is under an IOError?

Reply:
Account is actually created despite email failing to send verification token.

Reply:
Hmm I see, if this is for sending the user the verification token, and we failed to send them a verification token, then isn't their account unuseable? Do we have a mechanism for resending or dealing with this issue?

Reply:
Yes - I believe a resend-verification token was created but not wired on the client side yet. So it is possible for resending.

The problem is that we depend on a 3rd party email service, and our email can block after a limited amount of sends. We recognize this issue but don't necessarily have a way to solve it yet.